### PR TITLE
Fix ci error

### PR DESCRIPTION
**Pull request by JingfeiPeng**
fixed ci tests on smarts side, still needs to figure out how to fix ci test on Ultra side

### CODE REVIEW

**Comment by JingfeiPeng** [comment]
To reproduce the failing CI test locally, create two venv, one based on smarts ci and one based on Ultra's CI:

```
python3.7 -m venv .first
. .first/bin/activate
pip install --upgrade pip
pip install --upgrade wheel
pip install --upgrade -r requirements.txt
pip install --upgrade -e .[train,test]
```
then `cd ultra` and run `pytest tests/test_train.py` the test passes

Create venv according to Ultra's CI:
```
cd ultra
python3.7 -m venv .second
. .second/bin/activate
pip install --upgrade --upgrade-strategy eager pip
pip install --upgrade -e .
```

then run `pytest tests/test_train.py` we can reproduce the numpy import error
```
.ultra_venv/lib/python3.7/site-packages/smarts/core/utils/pybullet.py:27: in <module>
    from pybullet import *
E   ImportError: numpy.core.multiarray failed to import
```

**Comment by JenishPatel99** [comment]
@JingfeiPeng I have managed to suppress the ultra ci error by upgrading the numpy package. A stackexchange thread seemed to give a solution here is the link [https://stackoverflow.com/questions/33859531/runtimeerror-module-compiled-against-api-version-a-but-this-version-of-numpy-is](url)

File: .github/workflows/ci-base-tests.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: SMARTS CI Base Tests
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - disabled
```
Comment (JingfeiPeng) [inline .github/workflows/ci-base-tests.yml:6]: this would disable the tests right? We probably want to fix it instead of disabling it

Comment (JenishPatel99) [reply .github/workflows/ci-base-tests.yml]: Yes, sorry I forgot to enable those tests after just testing the ultra-ci by itself. I have restored the others tests as well

**Review by Gamenot — APPROVED** [review]

**Comment by Gamenot** [comment]
We should merge this into both develop and master.

**Pull request merged**